ballerina-tests/http-advanced-tests/Ballerina.toml

```diff
@@ -1,17 +1,17 @@
 [package]
 org = "ballerina"
 name = "http_advanced_tests"
-version = "2.9.1"
+version = "2.10.0"
```

Comment:
Do we need the version bump for this change?

Reply:
Yes
We automatically commit these. Incase something breaks the build we know the last compatible versions from them.
This is actually the version of the test module which not that much important. But we commit this for consistency